vitals/BatteryCheck.java

```diff
@@ -0,0 +1,39 @@
+package vitals;
+
+public class BatteryCheck {
+	public float Temperature, StateOfCharge, ChargeRate;	 
+    public BatteryCheck(float temperature, float soc, float chargeRate)
+    {
+        this.Temperature = temperature;
+        this.StateOfCharge = soc;
+        this.ChargeRate = chargeRate;
+    }
+    
+    
+    public static void CheckLowBreach(BatteryConstants battery)
+    {
+        if ((battery.value > (battery.minimumLimit + battery.lowBreach)) && (battery.value<(battery.minimumLimit + battery.highBreach))){

```

Comment:
Check****************Breach functions should return true/false .....don't apply SwitchLanguage.*****Breach.... let Check******Breach function remain pure and one type of responsibility